### DISCUSSION TITLE
2.8-ea lock=0 behaviour change

### DISCUSSION THREAD

**Discussion by korniltsev** [async-profiler/async-profiler#584] [Q&A] [answered]
In 2.7 lock=0 did not start LockTracer
In 2.8-ea lock=0 starts LockTracer with threshold=0
Is this an intended change? 

Probably changed here: https://github.com/jvm-profiling-tools/async-profiler/commit/0718a09e0e1a6fe461abc1a684f38149d43a6c37#diff-1af8dc685ba7f64f7123d4fd2e8e16262cf7a59c0a43ab95bf83fb7d77475b04L1272

**Comment by apangin** [comment, marked as answer]
Yes, it is.
